libcaf_core/caf/logger.hpp

```diff
@@ -64,8 +64,10 @@ class logger {
   struct event {
     event* next;
     event* prev;
+    int level;
+    std::string prefix;
```

Comment:
The prefix is all the logger-generated goop in front of the actual message. Only the log file contains this verbose representation. The console, which is meant for human consumption, merely contains the message.

For the console, I'm inclined to add the log facility/category filter, because often users of the CAF logging framework only want to see their own logs.